brunton_lab_to_nwb/brunton_lab_to_nwb.py

```diff
@@ -197,8 +198,11 @@ def run_conversion(
 
 
 def convert_dir(in_dir, n_jobs=1):
-    in_files = glob(os.path.join(in_dir, '*.h5'))
-    out_files = [os.path.splitext(x)[0] + '.nwb' for x in in_files]
+    all_files = Path(in_dir).iterdir()
+    all_data_files = [x.stem for x in all_files if ".h5" in x.suffix]
+    nwb_files = [x.stem for x in all_files if ".nwb" in x.suffix]
+    in_files = [str(in_dir / f"{x}.h5") for x in all_data_files if x not in nwb_files]
+    out_files = [str(in_dir / f"{Path(x).stem}.nwb") for x in in_files]
```

Comment:
Would be better to have an `overwrite=True` option on this function

Reply:
Sure, I'll add that

Reply:
Added